warp/examples/benchmarks/benchmark_sparse_marching_cubes.py

```
@@ -0,0 +1,279 @@
+        dense_ms = None
+        dense_tris = None
+        try:
+            dv, di = dense_extract(dense_field, origin, root_width, depth)
+            wp.synchronize_device(device)
+            dense_tris = len(di) // 3
+            del dv, di
+            dense_ms = (
+                time_call(lambda depth=depth: dense_extract(dense_field, origin, root_width, depth), device, args.iters)
+                * 1e3
+            )
+        except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)